### PR TITLE
Created add new test modal.

### PR DESCRIPTION
**Pull request by moustf**
- Created add new test modal.
- Styled the modal.
- Imported the modal in the app component.
---

Screenshot from 2022-10-25 11-55-22

### CODE REVIEW

**Review by Bas-Shiekh — DISMISSED** [review]

**Review by mahmoudhammad309 — APPROVED** [review]

**Review by Aaya-Elsharief — DISMISSED** [review]

File: client/src/components/ClassTests/AddTest/AddTest.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        hideClass: {
+          popup: "animate__animated animate__fadeOutUp",
+        },
+      });
+      source.cancel();
+    } catch (error: any) {
+      Swal.fire({
+        icon: "error",
+        title: "Oops...",
+        text: "Something went wrong!",
```
Comment (Raghad-Mezied) [inline client/src/components/ClassTests/AddTest/AddTest.tsx:63]: Add the error message you receive from the back

File: client/src/components/ClassTests/AddTest/AddTest.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      await Swal.fire({
+        title: "The test is added successfully!",
+        showClass: {
+          popup: "animate__animated animate__fadeInDown",
+        },
+        hideClass: {
+          popup: "animate__animated animate__fadeOutUp",
+        },
+      });
+      source.cancel();
```
Comment (Raghad-Mezied) [inline client/src/components/ClassTests/AddTest/AddTest.tsx:58]: Why are you canceling here? you have already done with your request

File: client/src/components/ClassTests/AddTest/AddTest.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    } catch (error: any) {
+      Swal.fire({
+        icon: "error",
+        title: "Oops...",
+        text: "Something went wrong!",
+        footer: '<a href="">Why do I have this issue?</a>',
+      });
+      source.cancel();
```
Comment (Raghad-Mezied) [inline client/src/components/ClassTests/AddTest/AddTest.tsx:66]: Same here

File: client/src/components/ClassTests/AddTest/AddTest.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+label {
+  font-size: 1.5rem !important;
+  color: #ADADAD !important;
+  font-weight: bolder;
+}
+
+input {
+  height: 60px !important;
+}
```
Comment (Raghad-Mezied) [inline client/src/components/ClassTests/AddTest/AddTest.css:78]: Don't do generic style

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Pull request merged**